analysis/paper/jcaa-paper.Rmd

```diff
@@ -36,9 +36,10 @@ library(Hmisc)
 
 # Introduction
 
-In this paper we investigate recent changes in the way computational archaeology is being done. Revolutions or paradigm-shifts in archaeology have been discussed to happen following theoretical discurses[@Clark_1993; @Härke_2002], but there has been an argument in philosophy of science whether revolutions in science are either ideas-driven or tool-driven [@Galison_1997; @Dyson_2000]. In this light we explore how many scientific disciplines have started to embrace methods to improve the reproducibility of their research and whether this change is part of a paradigm-shift. We focus on one set of tools for this change which is based on writing scripts in free and open source programming languages and review bibliometric evidence for this change in the sciences generally and in archaeology in particular. We evaluate how computational approaches to improving reproducibility and transparency in archaeology are mediated and transformed by digital approaches and test if the adoption of these new tools may be considered a revolution or some other process. 
+In this paper we investigate recent developments which facilitate computational archaeology to return to principles of the scientific method and whether this might be the beginning of a paradigm-shift. Revolutions or paradigm-shifts in archaeology have been discussed to happen following theoretical discourses [@Clark_1993; @Härke_2002], but there has been an argument in philosophy of science whether revolutions in science might not just be ideas-driven but also tool-driven [@Galison_1997; @Dyson_2000]. In this light we explore how Ecology and Archaeology have started to embrace methods which improve the reproducibility of their research. We focus on one set of tools for this change which is based on writing scripts in free and open source programming languages, exemplified by R, and the sharing of code. A review of bibliometric evidence showcases a strong development towards the use of R for Ecology and hints at the beginning of a similar development in archaeology. We thus evaluate how approaches to improving reproducibility and transparency in archaeology are mediated and transformed by digital approaches and propose these might be the sign of a tool-driven paradigm-change in archaeology. Acknowledging this is not a simple process, we offer a R-based tool to facilitates the task of creating a compendium which enables other researchers to reproduce the published results.
```

Comment:
I don't think we do see a paradigm shift here, so can we take out ' these might be the sign of a tool-driven paradigm-change in archaeology' and replace with  'these might be the sign of a tool-driven change in archaeology', so just remove the word paradigm?

Reply:
I guess I need to add some text that takes a stronger position on what we think is a paradigm shift, and what is not, and give some guidance to the reader about our views on that term. I could be more clear on this, I think

Reply:
Yeah, I think you're right about that. Also, should we expect a difference in a ideas-driven and a tools-driven paradigm change?

Reply:
Good question, yes, I think for tool-driven we probably wont have an 'incommensurate' change such as a paradigm change requires. The tool-driven change is more like a big step forward in how a research community does its work, but still pursuing the same goals as before, unlike the paradigm shift, when everything changes.